ihatemoney/forms.py

```diff
@@ -216,6 +217,9 @@ def save(self, bill, project):
         bill.payer_id = self.payer.data
         bill.amount = self.amount.data
         bill.what = self.what.data
+        tag = list(set(part[1:] for part in bill.what.split() if part.startswith('#')))
+        if tag:
+            bill.tag = tag[0]
```

Comment:
If I understand correctly, tags will be created like hashtags in tweets.

I think this could be mentioned somewhere in the bill creation form (tooltip?).
Don't you think they should be removed from the `what` column afterwards, to avoid duplication?

Reply:
We could definitely add a tooltip to the bill creation form! We weren't sure if that would clutter up the form too much.
As for removing it from the `what` column afterwards, we were thinking that people may also include the hashtag as part of their description like so: "#Rent for July" and in that case, we wouldn't want to remove it.

Reply:
OK, now I know that I don't use enough Twitter ;) I'm still using tags as optional fields like in blog post.

But you may be right, adding tags in another field is too complicated for this application's goal. So let's keep it like this for now.